pick.lic

```diff
@@ -708,48 +714,104 @@ class Pick
 
   def loot(box_noun)
     echo "Looting #{box_noun}..." if @debug
-    if DRC.bput("open my #{box_noun}", /You open/, /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
+    
+    if DRC.bput("open my #{box_noun}", /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
       echo 'Bug: Tried to loot locked box...'
       return
     end
-
-    if (@settings.fill_pouch_with_box || @settings.loot_specials.empty?)
+		
+    #pouch full, fill from lootpouch then try again      	
+    if Flags['full-lootpouch']
+      Flags.reset('full-lootpouch')
+      echo 'last match:, #{Regexp.last_match}'	 
+      echo 'last match0:, #{Regexp.last_match(0)}'	 
+      echo 'last match1:, #{Regexp.last_match(1)}'	       
+      case DRC.bput("fill my #{@settings.gem_pouch_adjective} #{@settings.gem_pouch_noun} with my #{@settings.autoloot_container}", /You open/, /is too full to fit another gem/, /is too full to fit any more gems/, /You'd better tie it up before putting/, /You'll need to tie it up before/)        
+      when /is too full to fit another gem/, /is too full to fit any more gems/
+        swap_out_full_gempouch
+        #fill pouch from container
+        DRC.bput("fill my #{@settings.gem_pouch_adjective} #{@settings.gem_pouch_noun} with my #{@settings.autoloot_container}", /You open/, /You fill your/)
+        DRC.bput("fill my #{@settings.gem_pouch_adjective} #{@settings.gem_pouch_noun} with my #{box_noun}", 'You fill your', 'You open your', 'What were you referring to', /any gems/, /too full to fit/)      
+        when /You'd better tie it up/, /You'll need to tie it up before/
+          #tie pouch once there is some gems in it
+          DRC.bput("tie my #{@settings.gem_pouch_adjective} #{@settings.gem_pouch_noun}", 'You tie', "it's empty?", 'has already been tied off') if @tie_gem_pouches
+          DRC.bput("fill my #{@settings.gem_pouch_adjective} #{@settings.gem_pouch_noun} with my #{@settings.autoloot_container}", /You open/, /is too full to fit another gem/, /is too full to fit any more gems/, /You'd better tie it up before putting/, /You'll need to tie it up before/)    		
+        end
```

Comment:
I believe this block should be un-indented to be in line with the case statement, as it's part of it.